### ISSUE TITLE
Tooltip brokes after changing range with nodata

### ISSUE THREAD

**Issue by mammuthus** [flant/grafana-statusmap#70]
```
grafana-statusmap version 0.1.1
Grafana v6.1.6 (commit: cf9cb45)
Datasource: Zabbix API 4.2.1

4 hosts
4 boolean metrics
```

Tooltip is okay if there is enough data in the range
![image](https://user-images.githubusercontent.com/1700286/58160678-98f2f800-7c87-11e9-8c87-d1f88b320bd8.png)

But if I select the range with nodata period, it brokes:
_(value should be 1)_
![image](https://user-images.githubusercontent.com/1700286/58160865-ec654600-7c87-11e9-8b30-042b3ca619db.png)
_(no tooltip at all)_
![image](https://user-images.githubusercontent.com/1700286/58160912-0010ac80-7c88-11e9-9c50-3548995ad3a2.png)

Settings
![image](https://user-images.githubusercontent.com/1700286/58161207-80371200-7c88-11e9-88b0-4302513edc2e.png)

**Comment by resuni** [comment]
I'm experiencing a similar issue where Statusmap is coloring a cell with a value of "1" as red and another point with a value of "1" as green. My color scale is 0.975-1. Setting "Display nulls" -> "As zero" rearranges what cells get colored red. In both cases, tooltips do not appear on a certain series of cells. 

I notice problems like this when I use statusmap with OpenTSDB as a data source and missing/null values are returned. Statusmap works flawlessly with MySQL as a datasource, but I've not tried this where MySQL returns null/missing values.

**Comment by diafour** [comment]
The problem with tooltip is fixed in #90. It is a part of v0.3.0 release.
